src/FOMObot/Helpers/Redis.hs

```diff
@@ -0,0 +1,51 @@
+module FOMObot.Helpers.Redis
```

Comment:
Looks like usage echoes this separation - prefs are all used in `FOMObot.Helpers.Bot` and `set/isDMChannel` is used in `FOMObot.App`.